mkosi

```diff
@@ -150,7 +150,7 @@ def btrfs_subvol_create(path, mode=0o755):
     os.umask(m)
 
 def btrfs_subvol_delete(path):
-    subprocess.run(["btrfs", "subvol", "delete", path], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True)
+    subprocess.run(["btrfs", "subvol", "delete", path], stdout=subprocess.DEVNULL, check=True)
```

Comment:
we invoke this on non-btrfs file systems, too, and we really don't want to show the error messages this causes to the user. If people run ext4 and this call hence necessarily fails they really shouldn't be alerted.

And do note that we try a number of different ways to remove a dir, one of them is this, and if it fails it doesn't really matter too much, we'll catch the error later on in a different way...